locales/ca/json.json

```diff
@@ -256,26 +256,26 @@
     "Gibraltar": "Gibraltar",
     "Go back": "Tornar",
     "Go Home": "Aneu a l'inici",
-    "Go to page :page": "Aneu a la pàgina de :page",
-    "Great! You have accepted the invitation to join the :team team.": "Genial! Que han acceptat la invitació a unir-se a la :team equip.",
+    "Go to page :page": "Aneu a la pàgina :page",
+    "Great! You have accepted the invitation to join the :team team.": "Genial! Heu acceptat la invitació a unir-se a l'equip :team.",
     "Greece": "Grècia",
     "Greenland": "Groenlàndia",
     "Grenada": "Granada",
     "Guadeloupe": "Guadalupe",
     "Guam": "Guam",
     "Guatemala": "Guatemala",
-    "Guernsey": "Atenes",
+    "Guernsey": "Guernsey",
```

Comment:
Add the value to the [`locales/ca/_excludes.json`](https://github.com/Laravel-Lang/lang/blob/main/locales/ca/_excludes.json) file, otherwise it will show up as not translated.